nginx/nginx.conf

```diff
@@ -26,6 +25,16 @@ server {
     add_header X-Request-Time $request_time;
     # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#Directives
     add_header Content-Security-Policy "default-src 'self'; img-src *; style-src 'self' 'unsafe-inline'; frame-ancestors 'none'; base-uri 'self'; form-action 'self'; report-uri https://recipeyak.report-uri.com/r/d/csp/enforce";
+    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy
+    add_header Referrer-Policy "strict-origin-when-cross-origin";
+    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options
+    add_header X-Content-Type-Options "nosniff";
```

Comment:
You need to fix this then:

https://github.com/recipeyak/recipeyak/blob/master/frontend/public/index.html#L28

since the type is only style, not `text/css`